account_consolidation/demo/company_a_demo.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="res.company" id="subsidiary_a">
```

Comment:
Akim, sorry but we need to demo data on teh odoo chart of account template. 
It is really a mess when you create a OCA demo instance